web/cobrands/peterborough/layout.scss

```diff
@@ -0,0 +1,101 @@
+@import "_colours";
+
+@import "../sass/layout";
+
+body.twothirdswidthpage .content .sticky-sidebar aside {
+    top: 10em;
+}
+
+#site-logo {
+    padding: 1em 0;
+}
+
+body.frontpage #front-main, 
+body.twothirdswidthpage #front-main, 
+body.fullwidthpage #front-main, 
+body.authpage #front-main {
+    text-align: left;
+    background-color: #fff;
+    border-top: 1px solid #e9e9e9;
+    color: $primary_b;
+    padding: 2.5em 0;
+}
+
+#front-main { 
+    #postcodeForm div {
+        margin: 0;
```

Comment:
Might not be needed due to flexbox changes in CE cobrand - I'll pull them out into separate PR to make this easier (will do same with alert index template refactoring as well I guess?)